pages/dao/[symbol]/index.tsx

```diff
@@ -266,14 +266,6 @@ const REALM = () => {
     setIsMultiVoting(false)
   }
 
-  const showMultiVote = useMemo(
```

Comment:
minor change after chatting sebastian -- batch voting should just be visible always. its not like gmail disables the batch operation UI just because you only have 1 email in the inbox.